docs/content/guide/dashboard.md

```diff
@@ -144,41 +144,86 @@ export default function(req, res) {
       displayedColumns: ["position", "name", "weight", "symbol"]
     }
   };
-  response.send([
+  res.send([
+    linedata,
     bubbledata,
     scatterdata,
-    linedata,
     piedata,
     radardata,
     doughnutdata,
     polarAreadata,
     bardata,
-    tabledata
-  ]);
+    tabledata]
+  );
 }
 ```
 
 And initialize dashboards on another function:
 
-```typescript
+```javascript
 // Add `@spica-devkit/dashboard` to your dependencies and import it
 import * as Dashboard from "@spica-devkit/dashboard";
-export default async function () {
+export default async function (req,res) {
   // Create an API key and initialize dashboards with it
-  Dashboard.initialize(<APIKEY>);
+
+  Dashboard.initialize("1i7z1kcoxehci");
```

Comment:
I am not sure if this is the current way of initializing the dashboard package.
@htuna07 knows better about this. Do you think that this code is still valid?

Also, can we replace the API key with a placeholder one? like so
```suggestion
  Dashboard.initialize("<YOUR_API_KEY_GOES_HERE>");
```
so that the user doesn't assume that this code works out of the box.

Reply:
We updated this initialize method like `Dashboard.initialize({apikey:"1i7z1kcoxehci"})` from version 0.5.0(latest) of @spica-devkit/dashboard package.

Reply:
@TolgaYigit then we have to update the documentation as well.